code/datums/wires/simple_wire.dm

```diff
@@ -0,0 +1,22 @@
+/datum/wires/simple_wire   //two bit wire type just to add assemblies to things that dont have specific wires
+	randomize = 0
+
```

Comment:
Yeah but there's nothing to attach the wire to without a wire... Unless you mean to attach it by calling the proc directly instead of using the wire UI.